backend/src/auth/auth.service.spec.ts

```diff
@@ -158,38 +158,28 @@ describe('AuthService', () => {
     });
 
     describe('validateAccessToken', () => {
-        it('userId에 해당하는 user가 존재하면 true를 반환해야 한다.', async () => {
-            jest.spyOn(usersService, 'findOne').mockResolvedValue({ id: 1 } as Users);
-
-            const result = await service.validateAccessToken(1);
-
-            expect(result).toEqual(true);
-        });
-
-        it('userId에 해당하는 user가 존재하지 않으면 false를 반환해야 한다.', async () => {
+        it('userId에 해당하는 user가 존재하지 않으면 404 error를 반환해야 한다.', async () => {
             jest.spyOn(usersService, 'findOne').mockRejectedValue(new NotFoundException());
 
-            const result = await service.validateAccessToken(1);
-
-            expect(result).toEqual(false);
+            await expect(service.validateAccessToken(1)).rejects.toThrow(NotFoundException);
         });
     });
 
     describe('validateRefreshToken', () => {
-        beforeEach(() => {
-            jest.spyOn(usersService, 'findOne').mockResolvedValue(mockUser);
-        });
-
-        it('oauthId에 해당하는 user의 refresh token과 현재 token이 일치하면 true를 반환해야 한다.', async () => {
-            const result = await service.validateRefreshToken(mockUser.refreshToken, mockUser.oauthId);
+        it('oauthId에 해당하는 user가 존재하지 않으면 404 error를 반환해야 한다.', async () => {
+            jest.spyOn(usersService, 'findOne').mockRejectedValue(new NotFoundException());
```

Comment:
```suggestion
        it('oauthId에 해당하는 user가 존재하지 않으면 NotFoundException를 던진다.', async () => {
            jest.spyOn(usersService, 'findOne').mockRejectedValue(new NotFoundException());
```
예외 처리를 할 때 어떤 상태의 에러를 표현 하기 보다는 `throw new NotFoundException`를 사용하기 때문에 "~를 던진다." 더 어울릴 것 같습니다.

Reply:
수정해서 반영했습니다!